perl_modules/EBI/FGPT/CheckSet/Curation.pm

```diff
@@ -1231,20 +1116,7 @@ sub check_features_match_array {
 			$self->add_missing_features( \%feature_info );
 		}
 	}
-
-	elsif ( !$identifiers && $acc =~ /^A-GEOD-/ ) {
-
```

Comment:
I understand the rationale for removing the DB call that was only for the sake of an error message. I may be missing something, but would it not be good to retain a more generic message that something has gone wrong rather than removing the warning entirely?

Reply:
Ah, yes. I was under the impression the "else" after this block would kick in in that case and show the error. It does have the same text. But I'm realising now that the first is a warning and the else is an error. 🤔